### PR TITLE
Added new Faded theme.

### PR DESCRIPTION
**Pull request by josuave**
Added new Faded theme.
Modified default Cairo theme to allow future themes to adjust

### CODE REVIEW

**Comment by dremin** [comment]
@josuave Should these be included with the install?

**Comment by josuave** [comment]
We can, at some point we would like to make a repo browser for themes and extensions. Until then we have to either include them OR make instructions

**Comment by dremin** [comment]
Going to merge this, we can add to the installer later. To start, we could have an optional install item for "additional themes" that adds these (and any others), but I want to make sure that author attribution is surfaced in the UI somewhere.

File: Cairo Desktop/Cairo Desktop/Cairo.xaml [hunk 1]
```diff
@@ -83,6 +84,8 @@
                       Direction="90"
                       Opacity="0.6"
                       BlurRadius="0" />
+    <Thickness x:Key="MenuBarBorderThickness" Right="1" />
```
Comment (dremin) [inline Cairo Desktop/Cairo Desktop/Cairo.xaml:87]: Pretty sure this should be bottom, not right. Can you adjust?

Comment (josuave) [reply Cairo Desktop/Cairo Desktop/Cairo.xaml:87]: Hmmm, ill take a look

**Pull request merged**